pom.xml

```diff
@@ -351,7 +296,13 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <source>${project.build.directory}/manifests</source>
+                    <urls>
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.cloudserviceaccountrequest.crd.yaml,
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.cloudservicesrequests.crd.yaml,
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.kafkaconnections.crd.yaml,
+                        https://raw.githubusercontent.com/redhat-developer/app-services-operator/main/olm/olm-catalog/rhoas-operator/${rhoas-model.version}/manifests/rhoas-operator.serviceregistryconnections.crd.yaml
+                    </urls>
+                    <downloadTarget>${basedir}/target</downloadTarget>
```

Comment:
we missed an `mkdirs` @manusa 😅 , on it 🙂

Reply:
😓